src/main/resources/reference.conf

```diff
@@ -2,10 +2,15 @@
 touchpoint.backend.environments {
   SANDBOX {
     stripe {
-      api.key {
-        public = "pk_test_Qm3CGRdrV4WfGYCpm0sftR0f"
-        secret = "dummy-key"
+      AU {
+        api.key.public = "pk_test_m0sjR1tGM22fpaz48csa49us"
+        api.key.secret = ""
       }
+      default {
+        api.key.public = "pk_test_Qm3CGRdrV4WfGYCpm0sftR0f"
+        api.key.secret = ""
+      }
+      api.version = "2017-08-15"
```

Comment:
Do we need the version here? and it is only added to SandBox, is this correct?

Reply:
Not sure if we need it or not, I'll remove it for now